wordpress/post_install.md

```diff
@@ -24,7 +24,7 @@ To simplify this configuration when the WordPress is installed an script is crea
 Use the following command to execute it when your WordPress pod reach running state.
 
 ```
- kubectl exec -it svc/wordpress -- sh -c /var/www/html/civo-init.sh
+ kubectl exec -it svc/wordpress -- bash -c /var/www/html/civo-init.sh
```

Comment:
_civo-init.sh_ is executable, so I think I tested it like this
```kubectl exec -it svc/wordpress -- /var/www/html/civo-init.sh```

On the other hand, this change looks good to me. Thanks for the fix.